src/pymorize/config.py

```diff
@@ -0,0 +1,225 @@
+"""
+This module defines the configuration hierarchy for the pymorize application, using
```

Comment:
Let me see if I can summarize all this briefly...

So the config manager is a way of loading some defaults for the attributes/keys of the config from some user defined default and establishing a hierarchy of how configs are (or are not) updated? Cool!

One day... when we have time to rewrite the esm-tools parser... one day... very far in the future... XD